web-common/src/components/forms/Select.svelte

```diff
@@ -18,15 +22,15 @@
 </script>
 
 <div class="flex flex-col gap-y-2">
```

Comment:
The Figma includes some placeholder text for these Select components: https://www.figma.com/file/Qt6EyotCBS3V6O31jVhMQ7/RILL?type=design&node-id=7116-382806&mode=design&t=8UeglpfHSPS7bIzz-0